src/ui/components/Button/index.js

```diff
@@ -1,13 +1,40 @@
+/* @flow */
 import makeClassName from 'classnames';
 import { oneLine } from 'common-tags';
 import * as React from 'react';
-import PropTypes from 'prop-types';
 
 import Link from 'amo/components/Link';
 import log from 'core/logger';
 
 import './styles.scss';
 
+export type ButtonType =
+  | 'neutral'
+  | 'light'
+  | 'action'
+  | 'cancel'
+  | 'confirm'
+  | 'alert'
+  | 'none';
+
+type Props = {|
+  buttonType: ButtonType,
+  children?: React.Node,
+  className?: string,
+  disabled: boolean | null,
```

Comment:
I think we should not have that, it should be `boolean` only

Reply:
It requires a change in the `UserProfile` (or `UserProfileEdit`) component.

Reply:
ah! I missed this above. I'll update..